packages/web/src/components/ui/PackagePreview/PackagePreview.tsx

```diff
@@ -139,38 +206,66 @@ export default function PackagePreview({ name, version, opened }: Props) {
+                      <div className={styles.statTitle}>MIT license</div>
+                    </Skeleton>
+                    <Skeleton>
+                      <div className={styles.statSubtitle}>freely distributable</div>
+                    </Skeleton>
+                  </>
+                ) : (
+                  <>
+                    <div className={styles.statTitle}>MIT license</div>
+                    <div className={styles.statSubtitle}>freely distributable</div>
+                  </>
+                )}
               </div>
 
               <div className={clsx(styles.stat, styles.statListItemSmall)}>
                 <div className={styles.statHeader}>
                   <Icon kind='rating' color='#8E8AA0' className={styles.statIcon} />
                   Rating
                 </div>
-                <div className={styles.statTitle}>
-                  385
-                  {/* or: <div className={clsx(styles.statRating, styles.statRatingRed)}> */}
-                  <div className={clsx(styles.statRating, styles.statRatingGreen)}>
-                    <Icon
-                      kind='ratingArrow'
-                      width={12}
-                      height={12}
-                      className={styles.statRatingArrow}
-                    />
-                    +4
-                  </div>
-                </div>
-                <div className={styles.statSubtitle}>out of 12 842</div>
+                {packageDetailsLoading ? (
+                  <>
+                    <Skeleton>
+                      <div className={styles.statTitle}>385</div>
+                    </Skeleton>
+                    <Skeleton>
+                      <div className={styles.statSubtitle}>out of 12 842</div>
+                    </Skeleton>
+                  </>
+                ) : (
+                  <>
+                    <div className={styles.statTitle}>
+                      385
+                      {/* or: <div className={clsx(styles.statRating, styles.statRatingRed)}> */}
+                      <div className={clsx(styles.statRating, styles.statRatingGreen)}>
+                        <Icon
+                          kind='ratingArrow'
+                          width={12}
+                          height={12}
+                          className={styles.statRatingArrow}
+                        />
+                        +4
+                      </div>
+                    </div>
+                    <div className={styles.statSubtitle}>out of 12 842</div>
+                  </>
+                )}
               </div>
 
               <div className={clsx(styles.stat, styles.statListItemLarge)}>
                 <div className={styles.statHeader}>
-                  <Icon kind='dependency' color='#8E8AA0' className={styles.statIcon} />4 Dependency
+                  <Icon kind='dependency' color='#8E8AA0' className={styles.statIcon} />
+                  {!packageDetailsLoading && 4} Dependency
```

Comment:
Looks weird. Maybe hide/skeletonize the whole header?

Reply:
Agreed, but according to design you already can see site address and its favicon/logo while loading. Package number just doesn't show in skeleton, it's just shows up eventually when loading is completed:

![image](https://user-images.githubusercontent.com/2999208/189710028-e88ec837-d04a-4829-8f2f-7650d46c1be5.png)

Maybe it's a design mistake and we should add skeleton for packages as well?

Reply:
Looks like a design mistake indeed. Ok, I'll write it down on my design issues list.